src/ui/add-content/AddContentFormContainer.js

```diff
@@ -110,6 +113,9 @@ export const mapDispatchToProps = (dispatch, { intl, history, match: { params }
       );
     }
   }),
+  onSaveFromModal: () => { dispatch(setVisibleModal('')); dispatch(submit('editcontentform')); },
```

Comment:
can we use simpler names like onSave, onCancel, onDiscard?

Reply:
ok :) I wanted the variables be more self-explanatory 😄